scripts/ci/build.sh

```diff
@@ -2,17 +2,26 @@
 set -ex
 
 ROOTDIR="$(dirname "$0")/../.."
+# shellcheck source=scripts/common.sh
+source "${ROOTDIR}/scripts/common.sh"
+
+prerelease_source="$1"
+
 cd "${ROOTDIR}"
 
 # shellcheck disable=SC2166
 if [ "$CIRCLE_BRANCH" = release -o -n "$CIRCLE_TAG" -o -n "$FORCE_RELEASE" ]
 then
-    echo -n "" >prerelease.txt
+    [[ -n "$prerelease_source" ]] || fail "prerelease_source must not be set when building a release"
```

Comment:
As mentioned in https://github.com/ethereum/solidity/pull/13581#discussion_r1019728963, let's remove this after all and ignore `prerelease_source` when building a release, as you did originally.